### PR TITLE
Portfolio finish mobile version-all sections

### PR DESCRIPTION
**Pull request by TamaraDimov**
New sections added to the Html file:
- "Works" section
- "About myself" section 
Edit the CSS file with styles for the new contents

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -59,5 +60,178 @@ <h6 class="connect">LET'S CONNECT</h6>
...
+      <section class="decription">
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+      </section>
+      <section class="linkButtons">
+        <button class="link">html</button>
+        <button class="link">css</button>
+        <button class="link">javaScript</button>
```
Comment (Nyame-Wolf) [inline portfolio.html:89]: I suggest you put all the `<div class="mainContainer">` in one container and use a `section `tag on them with `display :grid` to align all the cards and then use display flex for all the individual containers. This was part of the project requirements. This means that for each `<div class="mainContainer">` that you have, will be styled using `display:flex` and the overall container will be a `<section>` that has `display:grid`

File: portfolio.html [hunk 2]
```diff
@@ -59,5 +60,178 @@ <h6 class="connect">LET'S CONNECT</h6>
...
+      </section>
+      <div class="seeMore">
+        <button class="more">See Project</button>
+      </div>
+    </div>
+    <!-- Portfolio "about me" section -->
+    <!-- ABOUT ME 1 PAGE -->
+    <div class="mainContainerAboutme">
```
Comment (Nyame-Wolf) [inline portfolio.html:157]: Kindly use `<section>` for this. This is semantically correct and screen reader are able to read your code easily. Each individual area of your project should be a section on its own

File: portfolio.html [hunk 3]
```diff
@@ -59,5 +60,178 @@ <h6 class="connect">LET'S CONNECT</h6>
...
+      <div class="seeMore">
+        <button class="more">See Project</button>
+      </div>
+    </div>
+
+    <!-- Third page  -->
+
+    <div class="mainContainer">
+      <section class="snapshoot">
+        <img src="img/Snapshoot Portfolio3.png" alt="Snapshoot_portfolio" />
```
Comment (Nyame-Wolf) [inline portfolio.html:100]: Kindly note that for each of your cards this should be a background image. use the `background url` css property. This images have no semantic meaning in your project.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @TamaraDimov  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css [hunk 4]
```diff
@@ -72,12 +82,307 @@ body {
...
+  width: 100vw;
+  height: auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  margin-bottom: 10%;
+  padding: 0;
+  margin-top: 5rem;
```
Comment (Ugboaja-Uchechi) [inline style.css:270]: - [ ] [OPTIONAL] Kindly give your about section a ` border-top-right-radius` as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 5]
```diff
@@ -72,12 +82,307 @@ body {
...
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  padding: 0;
+  background-image: url(img/Snapshoot\ Portfolio3.png);
+  background-repeat: no-repeat;
+  background-size: 80% 40%;
+  background-position-x: center;
+  background-position-y: 15px;
+}
```
Comment (Ugboaja-Uchechi) [inline style.css:159]: - [ ] [OPTIIONAL] Kindly make your text aligned vertically on the same line as your `img` 🙏 
- [ ] [OPTIONAL] Kindly make sure you give design the same spacing as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @TamaraDimov, 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**